RELEASE_NOTES.md

```diff
@@ -34,6 +34,10 @@ Update to Python 3
 1.1.1:
 Update server to support longer running tasks with longer timeouts.
 
+1.1.2:
```

Comment:
Why did a add a note for an older release? Don't you need to increment the version to 1.2.2 and then associate your changes with that?

Reply:
Hah I see why, during the period when this pr is waiting for review,  #41  is merged and updated the version.